external-crates/move/crates/move-compiler/tests/move_check/parser/byte_string_token_eof1.move

```diff
@@ -1,5 +1,5 @@
 module 0x42::M {
-    public fun bad_value(): vector<u8> {
+    public fun bad_value() {
         b"diem\"
```

Comment:
Bad value parses should return error expressions so that this still works.

Reply:
If/when you bail out of the function due to the stop set, you should fill the function body in with `UnresolvedError`